servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcExecutionStrategies.java

```diff
@@ -45,11 +45,22 @@ public static GrpcExecutionStrategy defaultStrategy() {
      * A {@link GrpcExecutionStrategy} that disables all offloads.
      *
      * @return {@link GrpcExecutionStrategy} that disables all offloads.
+     * @deprecated Replaced with more descriptive {@link #offloadNever()}.
      */
+    @Deprecated
     public static GrpcExecutionStrategy noOffloadsStrategy() {
         return NEVER_OFFLOAD_STRATEGY;
     }
 
+    /**
+     * A {@link GrpcExecutionStrategy} that disables all offloads.
```

Comment:
Should the javadoc in this class be consistent with `HttpExecutionStrategies`?

Reply:
yes, I have improved them now.